MAUI/Pull-to-Refresh/pulltorefresh-events.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: PullToRefresh Events and Commands in .NET MAUI PullToRefresh control | Syncfusion
+description: Learn about PullToRefresh Events And Commands support in the Syncfusion .NET MAUI PullToRefresh (SfPullToRefresh) control and more.
+platform: MAUI
+control: SfPullToRefresh
+documentation: ug
+--- 
+
+# PullToRefresh Events in .NET MAUI SfPullToRefresh (SfPullToRefresh)
+
+There are three built-in events in the PullToRefresh control, namely:
+
+1. `Pulling`
+2. `Refreshing`
+3. `Refreshed`
+
+The `Pulling` event will be notified whenever the swipe gesture is started. This event will notify the listener each and every time until the refresh content height exceeds. When you release the gesture from pullable content, the `Refreshing` event will be triggered. Now, the refresh operation can be performed. Once the content is refreshed, set `SfPullToRefresh.IsRefreshing` to `false` to stop the animation. Once the animation is stopped, the `Refreshed` event will be triggered to notify that the refreshing is completed.
+
+## Pulling
+
+The `Pulling` event is triggered whenever you start pulling down on the `PullableContent` with `PullingEventArgs` that contains the following properties.
+
+* **Cancel** - You can cancel the pulling action based on the `Progress` value.
+* **Progress** - Gets the progress completion value.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+    <syncfusion:SfPullToRefresh x:Name="pullToRefresh" 
+                                PullingEvent="PullToRefresh_Pulling" />
```

Comment:
change event method name as "OnPullToRefreshPulling".
follow same for other events.

Reply:
changed all the event names